bash_scripting_101/bash_scripting_101.md

```diff
@@ -37,241 +39,443 @@ script: https://kit.fontawesome.com/83b2343bd4.js
+
+[[ `touch data`| `mkdir data` | (`cd data`) ]]
+***
+<div class = "answer">
+You must first make the directory `data` and then move into that directory with `cd data`.
+
+The command `touch data` will create a file named `data`, not a directory that can contain files.
+</div>
+***
+
+
+2. Now that you have created the files `data_1` you want to put some content into it. How would you make the file `data_1` contain the text `sample 1: 45`?
+
+[[echo "sample 1: 45" >> data_1]]
+<script>
+let input = "@input".trim();
+  input == "echo \"sample 1: 45\" >> data_1" || input == "echo sample 1: 45 >> data_1" || input == "echo \"sample 1: 45\" > data_1" || input == "echo sample 1: 45 > data_1";
+</script>
+***
+<div class = "answer">
+The command `echo "sample 1: 45" >> data_1` will add `sample 1: 45` as a new line at the end of the current contents of the file `data_1`.
+
+Since there wasn't anything in `data_1` to begin with, you will get the same outcome if you use a single `>` to replace the entire file with `sample 1: 45`.
+
+The quotes around the text can also be safely omitted in this case, but that isn't a good habit to get into since you may need them later as you progress in your bash scripting.
+</div>
+***
 
-- Binary or Executable File: A file that causes a computer to follow a series of tasks represented in encoded instructions. It should be noted that .bin is the preferred Mac extension and .exe is the preferred Windows extension.
-- Unix: Widely used class of computer operating systems which support multitasking and multiusers.
-- Linux: Open source operating system modeled on Unix. Bash is the most common shell for Linux.
 
 ## Additional Resources
 
 - [Brief Illustration of the Difference between Shell and Kernel](https://www.geeksforgeeks.org/difference-between-shell-and-kernel/)
-- [Exhaustive Wiki of Linux Filesystem Hierarchy](https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/index.html)
-- [Reinforce Your New Knowledge through this Learing the Shell Page](https://linuxcommand.org/lc3_learning_the_shell.php)
-- [Unix Command Line I Arcus Education Webinar](https://digitalrepository.chop.edu/commandline_computingtools/3/)
-- [Unix Command Line II Arcus Education Webinar](https://digitalrepository.chop.edu/commandline_computingtools/2/)
-- [Intermediate Bash Scripting Arcus Education Webinar](https://digitalrepository.chop.edu/commandline_computingtools/1/)
 
-## Feedback
+- [Reinforce Your New Knowledge through this Learning the Shell Page](https://linuxcommand.org/lc3_learning_the_shell.php)
 
-_5 minute survey_
+### The bash manual
 
-At the beginning of this module we described the following goals:
+The bash manual is available to you whenever you are in a bash shell.  Entering `man command_name` reveals the documentation for the function `command_name`.  This can be useful if you think a command might be useful and want to learn more about it, or have forgotten how to use a command or what its options are.
```

Comment:
```suggestion
The bash manual is available to you whenever you are in a bash shell.  Entering `man command_name` reveals the documentation (`man` is short for "manual") for the function `command_name`.  This can be useful if you think a command might be useful and want to learn more about it, or have forgotten how to use a command or what its options are.
```

Also include info here about how to exit and advance in man pages

Reply:
good call!